packages/tupaia-web/src/Routes.tsx

```diff
@@ -53,12 +53,14 @@ const UserPageRedirect = ({ modal }: { modal: MODAL_ROUTES }) => {
  *
  * **/
 export const Routes = () => {
-  const { isLoading } = useUser();
+  const { isLoading, isFetched } = useUser();
+
+  const showLoader = isLoading && !isFetched;
```

Comment:
When there was an error in the user endpoint, isLoading was switching back and forth from true to false and getting stuck in an infinite loop. Adding the isFetched check, breaks the circuit.